### PR TITLE
chore: Update sql query ending show into consistent drizzle notation

### PR DESCRIPTION
**Pull request by JacksonMeade**
For maintainability and consistency, this PR converts the pure SQL query used on the endshow endpoint to drizzle notation. It also adds a 'started show' notification when a DJ starts a new show, with the timestamp included.

### CODE REVIEW

File: src/services/flowsheet.service.ts [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-  rotation,
-  library,
+  ShowDJ,
   artists,
-  DJ,
   djs,
-  FSEntry,
+  flowsheet,
+  library,
+  // NewShow,
```
Comment (jakebromberg) [inline src/services/flowsheet.service.ts:14]: Is this intentionally left here? If so, could you add a todo comment for what needs to happen in order to uncomment this?

File: src/services/flowsheet.service.ts [hunk 2]
```diff
@@ -164,26 +171,29 @@ const createJoinNotification = async (id: number): Promise<FSEntry> => {
...
+  await Promise.all(remaining_djs.map(async (dj) => {
+    await db.update(show_djs).set({ active: false }).where(eq(show_djs.dj_id, dj.dj_id));
+    if (dj.dj_id === primary_dj_id) return;
+    await createLeaveNotification(dj.dj_id);
+  }));
+
+  const dj_information = (await db.select().from(djs).where(eq(djs.id, primary_dj_id)).limit(1))[0];
+
+  await db.insert(flowsheet).values({
+    message: `End of Show: DJ ${dj_information.dj_name} left the set at ${new Date().toLocaleString('en-US', { timeZone: 'America/New_York' })}`
```
Comment (jakebromberg) [inline src/services/flowsheet.service.ts:192]: Is there any way to handle the timezone of the client? If I were to do a remote set from Seattle, would it make sense for me to see my own timezone?

Comment (JacksonMeade) [reply src/services/flowsheet.service.ts:192]: I can write that into the frontend, but for flowsheet-as-legal-FCC-document purposes, what do we want written to the backend?

Comment (jakebromberg) [reply src/services/flowsheet.service.ts:192]: That's up to you. Most systems default to UTC.

**Review by AyBruno — CHANGES_REQUESTED** [review]
Generally I've been opting to use the 'CURRENT_TIMESTAMP' Postgres function/variable/magic. Let's stick to that when inserting dates to stay consistent.

Additionally the sql query that was being used only requires one back and forth trip to the database. I know the ORM is nice, but for more complex queries like this I've been going with prepared sql queries  to offload work and network traffic to the db.

**Pull request merged**